File: C++/tugas1/1.1(hufi_beli_makan).cpp
#include<iostream>
 using namespace std;
  int main() {
    int a;
    cin >> a;
    if (a%5 != 0) {
      cout << "Harganya jelek :(";
      return 0 ;
        }
    int b = a/250;
    cout << b << " llembar 250 ribuan"<< endl;
    int c = (a%250)/100;
    cout << c << " llembar 100 ribuan"<< endl;
    int d = ((a%250)%100)/50;
    cout << d << " llembar 50 ribuan"<< endl;
    int e = (((a%250)%100)%50)/10;
    cout << e << " llembar 10 ribuan"<< endl;
    int f = ((((a%250)%100)%50)%10)/5;
    cout << f << " llembar 5 ribuan";
  }

File: Python/Kuliah/Daspro/tugas 1/1.1(HufiBeliMakan).py
n = int(input())
a = n//250
b = (n%250)//100
c = ((n%250)%100)//50
d = (((n%250)%100)%50)//20
e = ((((n%250)%100)%50)%20)//10
f = (((((n%250)%100)%50)%20)%10)//5
# g
an = str(a)
bn = str(b)
cn = str(c)
dn = str(d)
en = str(e)
fn = str(f)

if n%5 !=0:
    print("Harganya jelek :(")
    StopIteration

print ( an + ' ' + 'lembar 250 ribuan')
print ( bn + ' ' + 'lembar 100 ribuan')
print ( cn + ' ' + 'lembar 50 ribuan')
print ( dn + ' ' + 'lembar 20 ribuan')
print ( en + ' ' + 'lembar 10 ribuan')
print ( fn + ' ' + 'lembar 5 ribuan')